### PR TITLE
Better messages for stalker when person nick is unique

### PR DESCRIPTION
**Pull request by daniel1302**

### CODE REVIEW

File: plugins/stalker.py [hunk 1]
```diff
@@ -131,10 +132,16 @@ def stalk(self, sender_nick, args, **kwargs):
         for x in all_nicknames:
             if nick in x: result.update(x)
 
-        if nick in result: result.remove(nick)
+        removed_nick = False
```
Comment (pingwindyktator) [inline plugins/stalker.py]: I dont like the idea of introducing additional variable here. I bet there are more straightforward ways to do this.

File: plugins/stalker.py [hunk 2]
```diff
@@ -131,10 +132,16 @@ def stalk(self, sender_nick, args, **kwargs):
...
+        if nick in result:
+            result.remove(nick)
+            removed_nick = True
 
         if result:
             self.bot.say(f'other nicks of {nick}: {", ".join(result)}')
+        elif removed_nick:
+            msgs = ['Gods believe, that %s is unique in our world.', 'We have seen %s lately.', '%s does not have multi accounts :(', ]
```
Comment (pingwindyktator) [inline plugins/stalker.py]: I would like to have one, specific answer in that case. "I don't know other %s nicks" maybe?

**Review by pingwindyktator — CHANGES_REQUESTED** [review]
Please see line comments.

**Comment by daniel1302** [comment]
@pingwindyktator I updated the code :)

**Review by pingwindyktator — APPROVED** [review]

**Pull request merged**